charts/seed-monitoring/charts/grafana/dashboards/operators/kubernetes-deployments-dashboard.json

```diff
@@ -342,8 +342,7 @@
       "tableColumn": "{deployment=\"aws-lb-readvertiser\", job=\"kube-state-metrics\", namespace=\"kube-system\", type=\"seed\"}",
       "targets": [
         {
-          "expr": "max(kube_deployment_spec_replicas{instance=\"kube-state-metrics\",deployment=\"$deployment_name\"}) without (instance, pod)",
-          "interval": "",
+          "expr": "max(kube_deployment_spec_replicas{deployment=\"$deployment_name\"})",
```

Comment:
The changes in this file seem to be wrong rebase after https://github.com/gardener/gardener/pull/2428.